### DISCUSSION TITLE
Security Onion web console access

### DISCUSSION THREAD

**Discussion by Yawndy** [Security-Onion-Solutions/securityonion#2802] [Unsupported Versions]
I'm fairly new to security onion and this is my first time using it. I've tried installing security onion 2.3.10 through the ISO image. I've finished the installation and everything checks out OK for _sudo so-status_ but I can't seem to access the web console through another VM on the same network. 
I've tried allowing my entire subnet of 172.16.1.0/24 for each of the options in _sudo so-allow_ but it still doesn't work. Does anybody have any ideas why this isn't working? When I access the website it says "This site can't be reached".
Any help is greatly appreciated!

**Comment by weslambert** [comment]
Please try installing [2.3.21 ](https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md)(latest) and see if you continue to have issues.

**Comment by innovate-support** [comment]
Could be a number of things.  During the setup there's a section that specifically asks you how you intend to access the web interface.  It warns about strict cookie restrictions.  Did you specify an IP, or hostname, or FQDN?  Also, is the 'other VM' that you're accessing SO from able to see SO on the network, aka ping or nmap it?  You may have a simple firewall blocking issue that's outside of SO.  For my SO Manager I use FQDN and use 0.0.0.0/0 in order to allow any to access it since my team is remote and working from home with changing source IP's.  This may not be what you want to do, but does work.

**Reply by defensivedepth** [reply]
@innovate-support ++ on the IP or hostname. If hostname, can you resolve the hostname to the IP on the system you are using the browser on?

**Comment by AndriiStepovyi** [comment]
@innovate-support +++ >sudo so-allow 
choose a) for analyst and provide an ip to "allow"
#will pinhole the firewall and allow the ip to be accessed on Manager/SOC/Nginx
Rest does fall into setup specifics- should be on the same subnet for simplicity
